ush/python/ocean/storm_HeatFlux.py

```diff
@@ -98,166 +98,229 @@ def ZoomIndex(var,aln,alt):
 if tcid[-1].lower()=='w':
-   nprefix=model.lower()+tcid.lower()+'.'+cycle+'.hafs_hycom_hwp30'
+   nprefix = aprefix + '.hafs_hycom_hwp30'
 if tcid[-1].lower()=='c':
-   nprefix=model.lower()+tcid.lower()+'.'+cycle+'.hafs_hycom_hcp70'
+   nprefix = aprefic + '.hafs_hycom_hcp70'
 
-aprefix=storm.lower()+tcid.lower()+'.'+cycle
-atcf = aprefix+'.trak.'+model.lower()+'.atcfunix'
+#atcf = aprefix+'.trak.'+model.lower()+'.atcfunix'
+atcf = glob.glob(COMOUT+'/*.atcfunix.*')[0]
 
 # ------------------------------------------------------------------------------------
 # - preprocessing: subset and convert wgrib2 to netcdf
 
 #
-Rkm=500 	# search radius
+Rkm = 500 	# search radius
 
-scrubbase='./tmp/'
+scrubbase = './tmp/'
 
-part=nprefix.partition('.'+model.lower()+'_')[0]
-nctmp=os.path.join(scrubbase,part)
+part = nprefix.partition('.'+model.lower()+'_')[0]
+nctmp = os.path.join(scrubbase,part)
 if os.path.isdir(nctmp):
    shutil.rmtree(nctmp)
-p=Path(nctmp)
+p = Path(nctmp)
 p.mkdir(parents=True)
 
 # track
 adt,aln,alt,pmn,vmx=readTrack6hrly(atcf)
-#if tcid[-1].lower()=='l' or tcid[-1].lower()=='e':
-#    aln=[-1*a + 360. for a in aln]
 
 #afiles = sorted(glob.glob(os.path.join(COMOUT,nprefix+'phyf*.nc')))
 afiles = sorted(glob.glob(os.path.join(COMOUT,'*'+model.lower()+'prs.synoptic*.grb2')))
-afiles=afiles[::2]   # subset to 6 hourly intervals
-flxvar=':(LHTFL|SHTFL):surface:'
+afiles = afiles[::2]   # subset to 6 hourly intervals
+flxvar = ':(LHTFL|SHTFL):surface:'
 for k,A in enumerate(afiles):
-    fhr=int(A.partition('.0p03.')[-1][1:4])
+    fhr = int(A.partition('.0p03.')[-1][1:4])
     if fhr==0:
-      xvars=flxvar+'anl:'
+      xvars = flxvar+'anl:'
     else:
-      xvars=flxvar+"%g"%(fhr)
+      xvars = flxvar+"%g"%(fhr)
    
-    ncout=os.path.join(nctmp,'heatflx_f'+"%03g"%fhr+'.nc')
-    cmd='sh ./xgrb2nc.sh '+'"'+xvars+'"'+' '+A+' '+ncout
+    ncout = os.path.join(nctmp,'heatflx_f'+"%03g"%fhr+'.nc')
+    cmd = 'sh ./xgrb2nc.sh '+'"'+xvars+'"'+' '+A+' '+ncout
     os.system(cmd)
 
-nfiles=sorted(glob.glob(os.path.join(nctmp,'heatflx_*.nc')))
+nfiles = sorted(glob.glob(os.path.join(nctmp,'heatflx_*.nc')))
 del afiles
 
-xnc=xr.open_mfdataset(nfiles)
-xii,yii=ZoomIndex(xnc.isel(time=[0]),aln,alt)
-xindx=xii[::2]
-yindx=yii[::2]
+xnc = xr.open_mfdataset(nfiles)
+xii,yii = ZoomIndex(xnc.isel(time=[0]),aln,alt)
+xindx = xii[::2]
+yindx = yii[::2]
 
-var1=xnc['LHTFL_surface'].isel(longitude=xindx,latitude=yindx)
-var2=xnc['SHTFL_surface'].isel(longitude=xindx,latitude=yindx)
+varr1 = xnc['LHTFL_surface'].isel(longitude=xindx,latitude=yindx)
+varr2 = xnc['SHTFL_surface'].isel(longitude=xindx,latitude=yindx)
 
 del nfiles
 del xnc
 
-lns,lts=np.meshgrid(var1['longitude'],var1['latitude'])
-dummy=np.ones(lns.shape)
+lns,lts = np.meshgrid(varr1['longitude'],varr1['latitude'])
+dummy = np.ones(lns.shape)
+
+var_name1 = 'Latent Heat Flux'
+var_name2 = 'Sensible Heat Flux'
+var_name0 = 'Turbulence Heat Flux'
```

Comment:
Naming convention should be update to: "Enthalpy Flux", "Net Turbulent Heat Flux" or something similar.

Reply:
I agree with the naming convention. I will change to "Enthalpy Flux"